cache/lru_cache.h

```diff
@@ -150,9 +150,11 @@ class LRUHandleTable {
 
   // The table consists of an array of buckets where each bucket is
   // a linked list of cache entries that hash into the bucket.
-  uint32_t length_;
-  uint32_t elems_;
   LRUHandle** list_;
+  uint32_t length_;
+
+  // cache line alignment elems_ to avoid false sharing
+  CACHE_ALIGNED(CACHE_LINE_SIZE) uint32_t elems_;
```

Comment:
elems_ is frequently updated (i.e., invalidating the cacheline), hence it needs alignment. 
But list_ seems mostly a read-only variable. So it does not need alignment I feel.